fiat-roles/src/main/java/com/netflix/spinnaker/fiat/roles/UserRolesSyncer.java

```diff
@@ -31,6 +31,8 @@
 import com.netflix.spinnaker.fiat.providers.ResourceProvider;
 import com.netflix.spinnaker.kork.eureka.RemoteStatusChangedEvent;
 import com.netflix.spinnaker.kork.lock.LockManager;
+
+import java.util.*;
```

Comment:
Minor nit: Can we undo this wildcard import and add back the java.util imports as it were before?

Reply:
sure, done!